presto-spark-base/src/main/java/com/facebook/presto/spark/execution/PrestoSparkRequestErrorTracker.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spark.execution;
+
+import com.facebook.presto.server.RequestErrorTracker;
+import com.facebook.presto.spi.ErrorCodeSupplier;
+import io.airlift.units.Duration;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+import java.net.URI;
+import java.util.concurrent.ScheduledExecutorService;
+
+import static com.facebook.presto.spi.StandardErrorCode.NATIVE_EXECUTION_TASK_ERROR;
+
+@ThreadSafe
+public class PrestoSparkRequestErrorTracker
```

Comment:
Why do we need this new class? It does not seem to have additional functionality compared to its parent

Reply:
I don't quite understand. Can you clarify?

Reply:
Removed the `PrestoSparkRequestErrorTracker` and use the `RequestErrorTracker` directly.